iOS/pr-validation.yml

```diff
@@ -0,0 +1,60 @@
+# Xcode
+# Build, test, and archive an Xcode workspace on macOS.
+# Add steps that install certificates, test, sign, and distribute an app, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/xcode
```

Comment:
Where is `iOS/pr-validation.yml` being added to configuration so that it can run on any commit to this path?

Reply:
done, add the `paths` filter